lake-manifest.json

```diff
@@ -115,10 +115,10 @@
    "type": "git",
    "subDir": null,
    "scope": "leanprover-community",
-   "rev": "8b6048aa0a4a4b6bcf83597802d8dee734e64b7e",
+   "rev": "43bcb1964528411e47bfa4edd0c87d1face1fce4",
    "name": "aesop",
    "manifestFile": "lake-manifest.json",
-   "inputRev": "v4.15.0-rc1",
+   "inputRev": "master",
```

Comment:
That looks a bit suspicious to me. Is that supposed to have happened?

Reply:
Yes. Why were we pinning a version of aesop rather than using whatever version mathlib uses?